projects/torantie/proposal/README.md

```diff
@@ -0,0 +1,43 @@
+# Proposal: [Quadrilateral Mesh Smoothing with mean value weights]
+
+[torantie]
+
+[17.01.2021]
+
+## Abstract
+
+The aim of this project is to give the existing half-edge structure from our exercises quadrilateral support and implement the Laplacian smooth with mean value weights.
+
+## Motivation
+
+Our lecture already introduced the half-edge structure to us. This was intended as storage for a triangular mesh. Since the structure can also be used for quadrilateral meshes it could be interesting to extend the current implementation. Additionally, I want to focus on smoothing the different mesh structure. In the lectures and through our exercises we were introduced to Laplacian smoothing with uniform and cotan weight. I want to implement this smoothing and add the option of mean value weights, since i find the resulting mesh interesting.
+![see Floater](https://raw.githubusercontent.com/torantie/gp-ws2021/project-proposal/projects/torantie/proposal/assets/mean_value.png?raw=true)
+
+
+(see Floater)
+
+## Proposal
+
+I want to implement quadrilateral support for the existing half-edge structure we were using in our exercises and implement Laplacian smoothing with mean value weights.
+The project is mostly related to the content provided in lecture 1, 3 and 4. Specifically for the mean value weights of the Laplacian smooth, the paper of Floater describing mean value coordinates is relevant.
+This is relevant to geometry processing since it extends a data structure used for storing geometry and shows a different representation of a geometry object. Furthermore, it concerns itself with mesh smoothing which is relevant to decrease noise in the mesh (see lecture 3, slide 5).
+Half edge structures can be implemented in a multitude of ways (see Lutz Kettner and Kalle Rutanen) and the smoothing will be based on the paper from Floater and the lecture slides, therefore this project will not be based on an existing code solution.
+
+## Implementation
+
+The programming language will be JavaScript and three.js. These will be the milestones.
+
+1. Building/adjusting the half-edge connectivity.
```

Comment:
Probably the only change that you need to deal with is the loader where the way you construct a face.
Maintaining the connectivities seems to be the same(?). It is worth mentioning that a face does not have to be flat.